src/saga/templates.js

```diff
@@ -13,6 +20,22 @@ function* fetchTemplatesSaga() {
     yield put(setLoadingAction({ loading: false }))
 }
 
+function* fetchTemplatePreviewsSaga({ payload: { templateId, sessionId, inscriptionId } }) {
+    yield put(setTemplatesLoadingAction({ loading: true }))
+    const previews = yield call(callService, {
+        endpoint: `template/previews?templateId=${templateId}&sessionId=${sessionId}&inscriptionId=${inscriptionId}`,
```

Comment:
Later, you can take a look at https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams